data/functions.sql

```diff
@@ -214,7 +214,23 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql IMMUTABLE;
 
-CREATE OR REPLACE FUNCTION mz_calculate_road_level(highway_val text, railway_val text, aeroway_val text)
+CREATE OR REPLACE FUNCTION mz_calculate_ferry_level(way geometry)
+RETURNS SMALLINT AS $$
+DECLARE
+  way_length real := st_length(way);
+BEGIN
+  RETURN (
+    CASE
+      WHEN way_length > 51356 THEN  9
```

Comment:
We should show **some** ferries at zoom 8 (when OSM starts showing) as there are a few ferries in Natural Earth we show and only including them starting at zoom 9 would introduce a gap. The ferry between the North and South Islands in New Zealand is a good example. Should this be length of way based? Or curable in the data so if `ferry=primary` then promote it up from 9 to 8? There aren't very many of these, so a little data edits in OSM paired with a commit doesn't bother me.